tests/improver-extract/07-list-constraints.bats

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bats
+# -----------------------------------------------------------------------------
+# (C) British Crown Copyright 2018 Met Office.
```

Comment:
Can you make this (and any other licence info) Copyright 2017-2018, which will then match all the other updated copyright information. (Basically we are going to be using a range from when we started (2017) to wherever we are (2018).

Thanks.